src/pages/quote.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable arrow-body-style, react/jsx-one-expression-per-line */
```

Comment:
- [ ] Let's not disable eslint rules. These are here to help you write best code ever.